.github/workflows/prepare-release-branch.yml

```diff
@@ -28,14 +28,15 @@ jobs:
       - uses: actions/checkout@v3
 
       - name: Create release branch
+        shell: bash {0}
```

Comment:
can you add a comment here why this is needed?

Reply:
Was doing this to avoid failing the step if a command has a non-zero exit code. Realized I could fix that without this by just removing the `--exit-code` flag from the `git ls-remote` command so it returns without an exist code if it doesn't find the ref.